### DISCUSSION TITLE
--exec Commands Adding to Size?

### DISCUSSION THREAD

**Discussion by ProfessorManhattan** [slimtoolkit/slim#304] [Q&A]
Hey, is it possible that the commands I'm running with `--exec` are adding new files and increasing the size of the image? Is that what this command is used for: `--remove-file-artifacts`?

**Comment by kcq** [comment]
Yes, it is possible that the `--exec` commands create additional files. The `--include-new=false` should prevent it. The `--remove-file-artifacts` flag removes the command execution related files that get generated (like the container report file or the security profiles).
